lib/netbox_client_ruby/api/ipam/ip_address.rb

```diff
@@ -22,7 +23,8 @@ class IpAddress
       vrf: proc { |raw_data| Vrf.new raw_data['id'] },
       tenant: proc { |raw_data| Tenant.new raw_data['id'] },
       status: proc { |raw_data| STATUS_VALUES.key(raw_data['value']) || raw_data['value'] },
-      interface: proc { |raw_data| Interface.new raw_data['id'] }
+      interface: proc { |raw_data| Interface.new raw_data['id'] },
+      address: proc { |raw_ip| IP.new(raw_ip) }
```

Comment:
Out of curiosity, how are these `object_fields` going to be used? I'm curious to see all these procs in action :).